apps/web-giddh/src/app/contact/contact.component.ts

```diff
@@ -1701,16 +1701,22 @@ export class ContactComponent implements OnInit, OnDestroy {
     public tabChange(event: any): void {
         if (event?.tab?.textLabel === this.localeData?.customer) {
             this.tabSelected("customer");
-            this.availableColumnsCount = [];
-            this.showFieldFilter = {};
-            this.translationComplete(true);
+            this.resetColumns();
         } else if (event?.tab?.textLabel === this.localeData?.vendor) {
             this.tabSelected("vendor");
-            this.availableColumnsCount = [];
```

Comment:
`resetColumns` is not called here, please check

Reply:
As per discussion , this function will not called here .